active/Expander/Expander.md

```diff
@@ -0,0 +1,131 @@
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+~~~~
+Public enum ExpandDirection 
+{ 
+Down = 0 
+Up = 1 
+} 
+ 
+public class Expander : ContentControl 
+{ 
+Expander(); 
+ 
+public object Header {get;set;} 
+public DataTemplate HeaderTemplate { get;set; } 
+public DataTemplate HeaderTemplateSelector {get;set;} 
+ 
+public static readonly DependencyProperty HeaderProperty; 
+public static readonly DependencyProperty HeaderTemplateProperty; 
+public static readonly DependencyProperty HeaderTemplateSelectorProperty {get;} 
+ 
+public bool IsExpanded { get;set} 
+public ExpandDirection ExpandDirection { get;set;} 
+ 
+protected virtual void OnExpanded(); 
+protected virtual void OnCollapsed(); 
+ 
+public event Expanded; 
+public event Collapsed; 
+ 
+public static readonly DependencyProperty IsExpandedProperty; 
+public static readonly DependencyProperty ExpandDirectionProperty; 
+} 
+~~~~
+
+
+# Appendix
+<!-- Anything else that you want to write down for posterity, but 
+that isn't necessary to understand the purpose and usage of the API.
+For example, implementation details. -->
+
+# Inputs and Accessibility
+## UI Automation Patterns
+Expander will use a ExpandCollapsePattern.
+
+## Keyboard Navigation
+TBD
+
+# Open Question
+## General
+* Should this be named "Expander" or "ExpanderView"?
+
+## Input and Accessibility
+* Should the first tab-stop be the entire Expander or just the Header? WCT Expander has the Header as the first tab-stop.
+* Should the second tab-stop be the chevron/expansion icon? WPF has the first tab-stop as the Header, the second as just the chevron. 
```

Comment:
I don't think we should match the WCT expander here, when navigating the expander with narrator, the header content does not get announced. I think having the icon be a separate tab stop is the better idea.

Reply:
Other feedback I've heard is that the if the only action that users can take on the Header is to expand/collapse, the icon shouldn't be a separate tab-stop. This gets messy when there's tab-stop content inside the header, though.